wasm/src/lib.rs

```diff
@@ -206,6 +206,14 @@ mod tests {
         print_u64("r", (a * b).to_canonical_u64());
         assert_eq!(c.to_canonical_u64(), (a * b).to_canonical_u64()); // compare to the non-wasm32 mult
         assert_eq!(c.to_canonical_u64(), 8353554915440457809 as u64);
+
+        let a: u64 = 9223372034707292160;
+        let b: u64 = 9223372034707292161;
+        let a = F::from_canonical_u64(a);
+        let b = F::from_canonical_u64(b);
+        let c = mul_wasm32(a, b);
+        assert_eq!(c.to_canonical_u64(), (a * b).to_canonical_u64()); // compare to the non-wasm32 mult
+        assert_eq!(c.to_canonical_u64(), 4611686017353646080 as u64);
```

Comment:
is it the same as in line 192?

Reply:
ah, yes :sweat_smile:

Reply:
I've updated the PR and removed that. I've also added new notes.